pySLM2/runtime_benchmark/README.md

```diff
@@ -0,0 +1,62 @@
+pySLM2 GPU Acceleration via Tensorflow
+======================================
+The `runtime_benchmark` folder contains tests for users to run on their machines to assess GPU availability and performance. To check for GPU availability and CUDA compatibility, execute the `check_gpu.py` file.
+
+Runtime Benchmarking Cases
+--------------------------
+At users' convinience, we separated out CPU/GPU-only tests for two different yet representative hologram generation cases.
```

Comment:
can you make a table instead?

Also, for benchmarking, can you show average and standard deviation

Reply:
done. check next commit